utils/README.md

```diff
@@ -1,79 +1,72 @@
-# Description of help utilities
-
-## [generateTestCaseList.pl]
-
-This tool extracts all Test Case specifications and creates
-an Markdown table to be inserted in [Test Case README]. This
-table should be recreated at each Zonemaster release.
-
-Use:
-
-1. Remove Test Case table from [Test Case README] and save the file
-2. Do:
+# Document generation
+
+## Background
+
+The utility scripts described below are used to update or generat some documents
+in the public document tree.
+
+* The scripts should be run at each Zonemaster release and the resulting update
+  included.
+
+## Pre-requisite
+
+The scripts should be run on a computer where the lastest `development` branch
+of `zonemaster/zonemaster` has been cloned, and the equivalent version of
+`zonemaster/engine` has been **installed**.
+
+## Scripts
+
+* [generateTestCaseList.pl]
+  * This tool extracts all Test Case specifications and creates a Markdown table
+    to be inserted in [Test Case README].
+* [updateTestPlanReadme.pl]
+  * This tool extracts all Test Case specifications per Test Plan and creates
+    Markdown tables. The tables are automatically added to the Test Plan
+    README.md file.
```

Comment:
```suggestion
    README.md file which is located in the directory named after the Test Plan.
```
To make sure the reader understands that this script modifies `*/README.md` files.

Reply:
Fine, I'll accept that, but this is information mainly for ourselves.